docs/docs/filter-sort.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Filter and Sort
+layout: docs
+---
+Use the `RequestOptions` object to define filtering and sorting criteria for an endpoint,
```

Comment:
I expect I'll expand this/add another topic for all RequestOptions later. Filtering and sorting seemed like critical things people would want to do though so I started with those.